episodes/11-exercises.md

```diff
@@ -114,33 +142,95 @@ and that assesses their ability to do so.
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
-Well-designed exercises are one of the most valuable resources for an Instructor:
-time spent on this is well invested.
+## Demo of Writing an Exercise 
+
+Well-designed exercises are one of the most valuable resources for an instructor and 
+any time spent on this is well invested.
+
+<!--- will there be an example lesson that the trainer is working on ? -->
+
+To create an exercise in the Carpentries Workbench, 
+you can use colon-delimited sections called 'fenced divs'.
+In fact there are many types of boxes in the lesson infrastucture, called "special blockquotes", 
+that use fenced divs.
```

Comment:
```suggestion
In fact [there are many types of boxes in the lesson infrastucture that use fenced divs](https://carpentries.github.io/sandpaper-docs/episodes.html#callout-blocks).
```

@zkamvar is there a better place to link to than this e.g. a single place that lists all the valid classes of fenced div?

Reply:
Oh hey! Yes, it is semi-documented [in {pegboard} with `validate_divs()`](https://carpentries.github.io/pegboard/articles/validation.html#fenced-div-validation), but I need to create the actual component guide 🙃